tools/file_packager.py

```diff
@@ -805,15 +805,9 @@ def generate_js(data_target, data_files, metadata):
         var DB_VERSION = 1;
         var METADATA_STORE_NAME = 'METADATA';
         var PACKAGE_STORE_NAME = 'PACKAGES';
-        function openDatabase(callback, errback) {'''
-      if options.support_node:
-        code += '''
-            if (isNode) {
-              return errback();
-            }'''
-      code += '''
+        function openDatabase(callback, errback) {
           if (typeof indexedDB == 'undefined') {
-            throw 'using IndexedDB to cache data can only be done on a web page or in a web worker';
+            return errback('using IndexedDB to cache data can only be done on a web page or in a web worker');
```

Comment:
We do have test coverage for this since without this part of the change the `other.test_file_packager_separate_metadata` fails under node